zebra-rpc/src/methods/get_block_template_rpcs.rs

```diff
@@ -310,34 +310,32 @@ where
                 data: None,
             })?;
 
-            // The tip estimate may not be the same as the one coming from the state
-            // but this is ok for an estimate
-            let (estimated_distance_to_chain_tip, estimated_tip_height) = latest_chain_tip
-                .estimate_distance_to_network_chain_tip(network)
-                .ok_or_else(|| Error {
-                    code: ErrorCode::ServerError(0),
-                    message: "No Chain tip available yet".to_string(),
-                    data: None,
-                })?;
-
-            if !sync_status.is_close_to_tip() || estimated_distance_to_chain_tip > MAX_ESTIMATED_DISTANCE_TO_NETWORK_CHAIN_TIP {
-                tracing::info!(
-                    estimated_distance_to_chain_tip,
-                    ?estimated_tip_height,
-                    "Zebra has not synced to the chain tip"
-                );
-
-                return Err(Error {
-                    code: NOT_SYNCED_ERROR_CODE,
-                    message: format!("Zebra has not synced to the chain tip, estimated distance: {estimated_distance_to_chain_tip}"),
-                    data: None,
-                });
+            {
+                // The tip height may change between this call and the `ChainInfo` request,
+                // but this is ok for estimating distance to the network tip and logging debug info.
+                let (estimated_distance_to_chain_tip, tip_height) = latest_chain_tip
```

Comment:
This probably needs to be called local_tip_height, to distinguish it from the estimated distance to the network tip.

Reply:
Changed in 59ff30faef87153028c1ac1af9b6ab5631ffd0d8 in PR #5796.